content/docs/SUSHI/tips/_index.md

```diff
@@ -271,3 +271,34 @@ This IG provides [MyPatient] for patient information.
 
 {% include fsh-link-references.md %}
 ```
+
+## Choosing a SNOMED CT edition
+
+To specify a certain edition of SNOMED CT - for example to ensure terminology validation works correctly - do the following:
+
+1. Add a new `parameters` section to your `sushi-config.yaml`:
+```yml
+parameters:
+  path-expansion-params: Parameters-expansion.json
+```
+2. Create an `input/fsh/expansion.fsh` file. If you'd like to use the US edition of SNOMED, fill this in:
+```
+Instance: expansion
+InstanceOf: Parameters
+Description: "SNOMED CT expansion parameter"
+Usage: #definition
+* parameter[+].name = "system-version"
+* parameter[=].valueCanonical = "http://snomed.info/sct|http://snomed.info/sct/731000124108"
+```
+
+If you'd like to use another language, such as Swedish, choose this:
+```
+Instance: expansion
+InstanceOf: Parameters
+Description: "SNOMED CT Swedish expansion parameter"
+Usage: #definition
+* parameter[+].name = "displayLanguage"
+* parameter[=].valueCode = urn:ietf:bcp:47#se
+* parameter[+].name = "system-version"
+* parameter[=].valueCanonical = "http://snomed.info/sct|http://snomed.info/sct/45991000052106"
+```
```

Comment:
It might be helpful to also provide a little context for these magic parameters, so people know where to look to understand them better. Consider adding something like this to the end:
```md
_Note: The `system-version` and `displayLanguage` parameters correspond to the parameters of the same name in FHIR's [$expand](https://hl7.org/fhir/valueset-operation-expand.html) operation._
```